### DISCUSSION TITLE
Upgrading from 0.27.0 (2.10.2) to latest 1.6.1

### DISCUSSION THREAD

**Discussion by dwpurser** [netbox-community/netbox-docker#743] [Getting Help]
I tried following the upgrade instructions following **Updating from below 1.x.x to 1.x.x** and then **PostgreSQL Update**. However, after the upgrade it fails to start. I do not know how to pull a certain release to step update, can you assist with a possible procedure?

**Comment by Fyne5** [comment]
My experience: dump database and save it

Up PostgreSQL (docker-compose up -d postgresql) to new version compatible with docker-netbox version only, restore database which dumped before, then up others.

Of courses, you must upgrade follow order from lowest to highest: example 1.1 > 1.2 > ... > 2.0
